gitlab/cli.py

```diff
@@ -308,6 +308,12 @@ def _get_base_parser(add_help: bool = True) -> argparse.ArgumentParser:
         action="store_true",
         default=os.getenv("GITLAB_SKIP_LOGIN"),
     )
+    parser.add_argument(
+        "--no-mask-credentials",
+        help="Don't mask credentials in debug mode",
+        dest="mask_credentials",
+        action="store_false",
```

Comment:
Follow-up could be a default with `os.getenv()` but we need our little strtobool helper first :+1: